tools/bpf2c/btf_parser.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+#include <map>
+#include <stdint.h>
+#include <string>
+#include <vector>
+
+typedef struct _line_info
+{
+    std::string file_name;
+    std::string source_line;
+    uint32_t line_number;
+    uint32_t column_number;
+} line_info_t;
+
+typedef std::map<std::string, std::map<size_t, line_info_t>> btf_section_to_instruction_to_line_info_t;
+
+/**
+ * @brief Parse a .btf and .btf.ext section from an ELF file and Map of section
+ * name to map of offset to line information.
```

Comment:
Can't parse grammar.
```suggestion
 * @brief Parse a .btf and .btf.ext section from an ELF file.
```